### ISSUE TITLE
Do you guys plan to support more LLMs with available SSMs?

### ISSUE THREAD

**Issue by pmking1** [flexflow/FlexFlow#1232]
Do we plan to support more LLM with available SSMs? If you do have, what's the schedule？

**Comment by jiazhihao** [comment]
Yes, we are in the process of boost-tuning SSMs for additional LLMs and plan to release them in the next several months.

**Comment by shannonphu** [comment]
Which LLMs are on your roadmap? Are you considering any encoder-decoder models like T5 family?
